Assets/Scripts/InventoryAndItemsScripts/InventoryChecker.cs

```diff
@@ -113,5 +113,15 @@ private bool CheckHaveEquippedAtLeastOneOfTheseItems()
             return this.mustHaveEquippedAtLeastOneOfTheseItems.Contains
                             (PlayerInventory.Instance.CurrentlyEquippedItem);
         }
+
+        public bool MeetsCriteria()
+        {
+            return InventoryStateMeetsCriteria();
+        }
+
+        public void TestSub()
```

Comment:
any reason to leave this here? were you planning on writing a test?

Reply:
whoops, thanks, love it when I code while tired rip